<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>351</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>电压</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电流</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>11</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>13</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>14</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>15</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>16</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>电压</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电流</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>电压</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电流</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>电压</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电流</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>电压</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电流</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>电压</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电流</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_7">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>电压</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电流</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_8">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>电压</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电流</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>电压</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电流</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_10">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>电压</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电流</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_11">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>电压</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电流</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_12">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>电压</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电流</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_13">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>电压</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电流</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_14">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>电压</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电流</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_15">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>电压</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电流</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_16">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>电压</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电流</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_DevID">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>设备ID</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>采样类型</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>采样频率</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_SampleRte">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>服务器发送间隔(分)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_SendTime2erver">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>390</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>触发态发送间隔(秒)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_SendTimeDynamic">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>410</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label__Message">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>480</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hz</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>分</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>秒</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
